packages/trader/src/App/Containers/trade-settings-extensions.jsx

```diff
@@ -43,23 +43,14 @@ class TradeSettingsExtensions extends React.Component {
         populateSettingsExtensions(menu_items);
     };
 
-    componentDidMount() {
-        this.populateSettings();
-    }
+    React.useEffect(() => {
+        return () => populateSettingsExtensions(null);
+    }, [populateSettingsExtensions]);
```

Comment:
Amir jan, I don't think we need to run this function whenever populateSettingsExtensions update you can remove the props
```suggestion
    }, []);
```